mips3.asm

```diff
@@ -0,0 +1,21 @@
+# Omer Aldafari		
+# Assembly
+# 2/11/15
+# Part 3		
+			
```

Comment:
-5 Zero comments.  Please make sure to add comments next time, its easy way to NOT lose points